---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}


Postdoctoral researcher with 8+ years of experience in microbiology, RNA sequencing, and bioinformatics.  
Specialized in long-read transcriptomics using Oxford Nanopore platforms, with practical experience across multiple NGS technologies (Nanopore, PacBio, Illumina).  
Passionate about translating complex biological data into actionable insights for research and diagnostics.  
Strong background in bioinformatics pipelines for RNA and DNA sequencing projects.

Education
======
* Ph.D. in Biology, University of Regensburg, 2021
* Extracurricular IT education, 2016
* M.Sc. in Biology, [University Name], 2016
* B.Sc. in Biology, [University Name], 2013

Research Interests
======
* Development and optimization of NGS workflows
* Bioinformatics and computational RNA biology
* Nanopore sequencing technologies 
* Prokaryotic transcriptomics
* RNA processing and regulatory networks
* Thermal stress response in extremophiles

Work Experience
======
* 2021–Present: Postdoctoral Researcher
  * University of Regensburg, German Archaea Centre
  * Focus: Prokaryotic RNA biology, single-molecule sequencing, transcriptome architecture
  * Duties included: Development of Nanopore sequencing protocols, bioinformatics analysis, interdisciplinary project collaboration

* 2016–2021: Doctoral Researcher
  * University of Regensburg
  * Focus: Transcriptional and post-transcriptional processes in Archaea
  * Duties included: RNA isolation, library preparation, sequencing data analysis, genome annotation, pipeline development

Skills
======
* Expertise in Oxford Nanopore sequencing (native RNA, cDNA, DNA)
* Practical experience with PacBio and Illumina NGS platforms
* Development, optimization, and validation of NGS workflows
* Bioinformatics analysis:
  * Sequence alignment (Minimap2, BWA)
  * Transcriptome assembly (StringTie2, TAMA)
  * Variant calling (SAMtools, NanoVar)
  * RNA modification detection (Tombo, EpiNano)
* Programming and pipelines (R, Python, Bash)
* Data visualization 
* Version control (Git, GitHub)

NGS Platforms and Tools
======
* Nanopore: MinKNOW
* Galaxy bioinformatics platform
* Linux/Unix environment for data processing

Project Management
======
* Leading NGS-related projects from design to reporting
* Multi-team collaboration with biologists and bioinformaticians
* Training and mentoring of junior researchers and students

Publications
======
<ul>{% for post in site.publications reversed %}
  {% include archive-single-cv.html %}
{% endfor %}</ul>

Service and Leadership
======
* Co-supervision of Bachelor and Master thesis students
* Contributor to open-source bioinformatics repositories
Languages
======
* German (native)
* English (full professional proficiency)

